File: templates/tests-go/BREACH-003.go
package compliance

import (
	"testing"
)

func TestVerifyPIICategoriesDocumented(t *testing.T) {
	config := GetConfig()
	if len(config.DataLifecycle.PIICategories) == 0 {
		t.Error("PII categories must be documented")
	}
}

File: templates/tests-python/BREACH-003.py
import pytest


def test_verify_pii_categories_documented():
    """Verify PII categories are documented."""
    config = get_config()
    assert "data_lifecycle" in config
    assert "pii_categories" in config["data_lifecycle"]
    assert len(config["data_lifecycle"]["pii_categories"]) > 0
